modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -54,7 +57,9 @@ account when estimating the number of security groups required for the quota.
 file and image registry.
 
 * Although the quota does not account for load balancer VM resources, they must be
-considered when deciding the OpenStack deployment size.
+considered when deciding the OpenStack deployment size. Note just the
```

Comment:
"Although the quota does not account for load balancer VM resources, you must consider them when you decide on the {rh-openstack} deployment's size. In the end, the installation will have more than 50 load balancers; the clusters must be able to accommodate them."